public/js/pages/index.js

```diff
@@ -22,41 +22,27 @@ require(['jquery', 'base/ui', 'base/gallery', 'base/login'],
     });
 
     UI.select('#search-filter', function (val) {
-
       var makes = document.querySelectorAll('.make');
 
-      switch (val) {
-      case 'featured':
-        gallery.searchOptions.tags = {
-          tags: ['webmaker:featured']
-        };
-        gallery.searchOptions.sortByField = ['createdAt', 'desc'];
-        delete gallery.searchOptions.contentType;
-        break;
-
-      case 'popcorn':
-        gallery.searchOptions.tags = {
-          tags: ['webmaker:featured']
-        };
-        gallery.searchOptions.sortByField = ['createdAt', 'desc'];
-        gallery.searchOptions.contentType = 'application/x-popcorn';
-        break;
-
-      case 'thimble':
-        gallery.searchOptions.tags = {
-          tags: ['webmaker:featured']
-        };
-        gallery.searchOptions.sortByField = ['createdAt', 'desc'];
-        gallery.searchOptions.contentType = 'application/x-thimble';
-        break;
-
-      case 'appmaker':
-        gallery.searchOptions.tags = {
-          tags: ['webmaker:featured']
-        };
-        gallery.searchOptions.sortByField = ['createdAt', 'desc'];
-        gallery.searchOptions.contentType = 'Appmaker';
-        break;
+      gallery.searchOptions.tags = {
+        tags: ['webmaker:featured']
+      };
+
+      gallery.searchOptions.sortByField = [
+        'createdAt',
+        'desc'
+      ];
+
+      delete gallery.searchOptions.contentType;
```

Comment:
this feels odd. I'd save the delete for when absolutely necessary because it'll mess up the JS compiler, forcing it to retry figuring out the JIT shape.